<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Importer</string>
  </property>
  <layout class="QVBoxLayout" name="main_lay">
   <item>
    <layout class="QGridLayout" name="parameters_lay" columnstretch="0,1,0">
     <item row="0" column="1">
      <widget class="QComboBox" name="path_cmb">
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="path_browse_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="path_lbl">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="config_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="config_cmb">
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="config_lbl">
       <property name="text">
        <string>Config</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="config_scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="config_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>753</width>
        <height>509</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="config_lay">
       <item>
        <layout class="QHBoxLayout" name="renderer_lay" stretch="0,1">
         <item>
          <widget class="QLabel" name="renderer_lbl">
           <property name="text">
            <string>Renderer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="renderer_cmb"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="subfolders_chk">
     <property name="text">
      <string>Include Sub Folders</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="footer_lay" stretch="1,0">
     <item>
      <widget class="QProgressBar" name="main_prgbar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="main_btnbox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
